File: app/src/main/java/ru/melodin/fast/util/StringUtils.java
package ru.melodin.fast.util;

public class StringUtils {

    public static String unescape(String input) {
        return input
                .replace("&gt;", ">")
                .replace("&lt;", "<")
                .replace("&quot;", "\"")
                .replace("<br>", "\n")
                .replace("<br/>", "\n")
                .replace("amp;", "&");
    }

}

File: app/src/main/java/ru/melodin/fast/util/StringUtils.kt
package ru.melodin.fast.util

import android.text.Editable

object StringUtils {

    fun unescape(input: String): String {
        return input
            .replace("&gt;", ">")
            .replace("&lt;", "<")
            .replace("&quot;", "\"")
            .replace("<br>", "\n")
            .replace("<br/>", "\n")
            .replace("amp;", "&")
    }

    fun isEmpty(input: String?): Boolean {
        return input == null || input.trim().isEmpty()
    }

    fun isEmpty(input: Editable?): Boolean {
        return input == null || input.toString().trim().isEmpty()
    }
}